presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1441,16 +1441,20 @@ void PrestoServer::checkOverload() {
         : 0;
     const bool isMemOverloaded =
         (currentUsedMemoryBytes > overloadedThresholdMemBytes);
-    if (isMemOverloaded) {
-      LOG(WARNING) << "Server memory is overloaded. Currently used: "
+    if (isMemOverloaded && !isMemOverloaded_) {
```

Comment:
For workerOverloadedThresholdMemGb, should we define as a percentage of existing memory usage counter? Thanks!